src/utils/__tests__/index.ts

```diff
@@ -210,3 +214,18 @@ describe("getTokenIconPath", () => {
     })
   })
 })
+
+describe("createMultiCallContrat", () => {
+  it("correctly returns method call as an Object and not a Promise", () => {
+    const emptyAddress = "0x0000000000000000000000000000000000000000"
+    const helperContractMultiCall = createMultiCallContract<HelperContract>(
+      emptyAddress,
+      HELPER_CONTRACT_ABI,
+    )
+
+    expect(helperContractMultiCall).toBeInstanceOf(Contract)
+    expect(
+      helperContractMultiCall.gaugeToPoolAddress(emptyAddress),
+    ).toBeInstanceOf(Object)
```

Comment:
I wonder if everything in JS is actually an instance of Object. Might be more to the point to assert that it's not a function/promise, or that it has `ethcall/Call`-specific properties

Reply:
I added positive/negative key checks. I think it should be clear.